packages/datetime/test/dateTimePickerTests.tsx

```diff
@@ -101,6 +100,14 @@ describe("<DateTimePicker>", () => {
                 assert.isFalse(getSelectedDay().exists());
             });
         }
+
+        it("Passing the same value prop twice does not clear the selected date", () => {
+            const { root, getSelectedDay } = wrap(<DateTimePicker />);
+            assert.isTrue(getSelectedDay().exists());
+            root.setProps({ value: undefined });
```

Comment:
would be much clearer to use an actual `value` and pass it on line 105 too. 
`<DateTimePicker value={VALUE} />`